src/Makefile.am

```diff
@@ -69,6 +69,14 @@ libephymain_la_SOURCES = \
 	ephy-session.h				\
 	ephy-shell.c				\
 	ephy-shell.h				\
+	ephy-sync-crypto.c			\
```

Comment:
@mcatanzaro How to do that? I'm not sure I know how to write a Makefile.am from scratch for a new subdirectory.